pkg/daemon/daemon.go

```diff
@@ -270,33 +269,11 @@ const (
 )
 
 func (dn *Daemon) detectEarlySSHAccessesFromBoot() error {
-	journal, err := sdjournal.NewJournal()
+	o, err := exec.Command("journalctl", "-b", "-o", "cat", "MESSAGE_ID=" + sdMessageSessionStart).CombinedOutput()
```

Comment:
could we rename this var from `o` to something more descriptive?

Reply:
Sure, though I think it's better with comments like this to suggest a new name; have one in mind?

Reply:
`out`, `journalOutput`?

Reply:
ahh didn't want to be too prescriptive @cgwalters but point taken :)

journalOutput  wfm.